client/src/app/site/motions/services/motion-repository.service.ts

```diff
@@ -43,6 +44,16 @@ import { Mediafile } from 'app/shared/models/mediafiles/mediafile';
     providedIn: 'root'
 })
 export class MotionRepositoryService extends BaseRepository<ViewMotion, Motion> {
+
+    // The line length; comes from the config variable motions_line_length
+    private lineLength = 80;
```

Comment:
The default value of [motions_line_length](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/motions/config_variables.py#L84) is 90! 
@CatoTH Should we change it here? Maybe a better way: you can get the default value of the config instead hard coding?